### PR TITLE
Fixes build failure because of merging #12608

### PR DESCRIPTION
**Pull request by ajainarayanan**
Merging https://github.com/cdapio/cdap/pull/12608 PR caused builds to fail. This PR is intended to be merged after the schema editor changes which were missing in the release/6.2 branch.

Removing this change and will add it back in the Schema editor PR https://github.com/cdapio/cdap/pull/12638

Build: https://builds.cask.co/browse/CDAP-URUT327-1

### CODE REVIEW

**Comment by yukiej** [comment]
lgtm!

**Review by yukiej — APPROVED** [review]

**Comment by ajainarayanan** [comment]
Running locally seemed to pass.
```
      Spec                                              Tests  Passing  Failing  Pending  Skipped  
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  admin.spec.ts                            00:05        2        2        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  controlcenter.spec.ts                    00:31        1        1        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  fieldlevellineage.v2.spec.ts             02:25        3        3        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  lab.spec.ts                              00:23        5        5        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  login.spec.ts                             80ms        2        2        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  navbar.spec.ts                           01:49        7        7        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  nuxtour.spec.ts                          00:12        5        5        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  page.level.errors.spec.ts                01:39       15       15        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  pipeline.canvas.actions.spec.ts          00:41        3        3        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  pipeline.complexschema.spec.ts           00:43        1        1        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  pipeline.contextmenu.spec.ts             00:54        8        8        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  pipeline.joiner.spec.ts                  01:50        3        3        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  pipeline.multiselect.spec.ts             00:10        1        1        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  pipeline.plugin.multipleendpoints.s      00:52        2        2        -        -        - │
  │    pec.ts                                                                                      │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  pipeline.runtimeargs.spec.ts             03:20        8        8        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  pipeline.spec.ts                         00:10        1        1        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  pipeline.studio.spec.ts                  00:45        5        5        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  pipeline.upgrade.spec.ts                 00:39        3        3        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugin.upload.spec.ts                    00:07        2        2        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins.validation.spec.ts               01:14        4        4        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✖  systemdelay.spec.ts                      01:32        3        -        1        -        2 │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  widgets.CSVWidget.spec.ts                00:13        5        5        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  widgets.KeyValueDropdownWidget.spec      00:14        5        5        -        -        - │
  │    .ts                                                                                         │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  widgets.KeyValueWidget.spec.ts           00:14        5        5        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  wrangler.bigquery.spec.ts                00:55        7        7        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  wrangler.gcs.spec.ts                     00:49        7        7        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  wrangler.spanner.spec.ts                 00:51        7        7        -        -        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✖  1 of 27 failed (3%)                      23:28      120      119        1        -        2  
```
The System delay tests are failing because the feature is not cherry-picked entirely (https://github.com/cdapio/cdap/pull/12603). Once the PR is merged it should work.

Merging this to unblock build failure. 

Green build: https://builds.cask.co/browse/CDAP-URUT327-1

**Pull request merged**
